playbooks/roles/mattermail/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+- name: create mattermail group
+  group:
+    name: mattermail
+
+- name: create mattermail user
+  user:
+    name: mattermail
+    group: mattermail
+    home: "{{ MATTERMAIL_HOME }}"
+
+- name: Install git package
+  apt:
+    name: git
+    update_cache: yes
+
+- name: Install golang package
+  apt:
+    name: golang
+
+- name: build Mattermail binary
+  command: go get {{ MATTERMAIL_REPO }}
```

Comment:
Damn, there doesn't seem to be a module for this, which is surprising.

Reply:
Also maybe you want to use `args.creates` (see [these examples](http://docs.ansible.com/ansible/latest/modules/command_module.html#examples)) so that this doesn't always run? Same for any other `command` modules used here.

Reply:
Good tip!